src/v2/Apps/Partner/Components/PartnerViewingRooms/ViewingRoomsPaginated.tsx

```diff
@@ -0,0 +1,195 @@
+import { Box } from "@artsy/palette"
+import React, { useState } from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { PaginationFragmentContainer } from "v2/Components/Pagination"
+import { LoadingArea } from "v2/Components/LoadingArea"
+import { ViewingRoomsFragmentContainer } from "v2/Apps/Partner/Components/PartnerViewingRooms/ViewingRooms"
+import { useSystemContext } from "v2/System"
+import { useRouter } from "v2/System/Router/useRouter"
+import { ViewingRoomsPaginatedRendererQuery } from "v2/__generated__/ViewingRoomsPaginatedRendererQuery.graphql"
+import { ViewingRoomsPaginated_partner } from "v2/__generated__/ViewingRoomsPaginated_partner.graphql"
+import { SystemQueryRenderer as QueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { compact } from "lodash"
+
+interface ViewingRoomsProps {
+  relay: RelayRefetchProp
+  partner: ViewingRoomsPaginated_partner
+  scrollTo: string
+  eventTitle: string
+  offset: number
+  paramsPage?: number
+}
+
+const ViewingRoomsPaginated: React.FC<ViewingRoomsProps> = ({
+  eventTitle,
+  relay,
+  partner,
+  scrollTo,
+  offset,
+  paramsPage,
+}) => {
+  const {
+    match: { location },
+    router,
+  } = useRouter()
+  const [isLoading, setIsLoading] = useState(false)
+
+  if (!partner?.viewingRoomsList) {
+    return null
+  }
+
+  const {
+    viewingRoomsList: {
+      edges: viewingRooms,
+      pageInfo: { hasNextPage, endCursor },
+      pageCursors,
+    },
+    slug,
+  } = partner
```

Comment:
non-blocking: Small preference for splitting this into 3 lines:
```js
const { viewingRoomsList, slug } = partner
const { edges, pageInfo, pageCursors } = viewingRoomsList
const { hasNextPage, endCursor } = pageInfo
```
I find it easier when all available variables are on the left-most side of the line, but I understand the succinctness of how you did it!

Reply:
fixed!